fpsim/locations/data/senegal/method_mix_matrix_switch.csv

```diff
@@ -0,0 +1,43 @@
+"No","None",147.590019,"18-20",0.117916307064098,0.0206186571464565,0.40885532374652,0.178591588906835,0,0,0.134807476378196,0.107421816918392,0.0317888298395029,"grpNo18-20"
+"No","None",159.031648,"<18",0.0784528938541843,0.00610340779465481,0.288778262550609,0.431526126170811,0,0,0.17769448003205,0.0147844660454,0.00266036355229118,"grpNo<18"
+"No","None",670.346782,"20-25",0.215128897269772,0.0195634294847708,0.414374193266434,0.0798326410105747,0,0.00865990432993531,0.222205706060956,0.0290848923624728,0.0111503362150846,"grpNo20-25"
+"No","None",724.118463,">35",0.239070149216731,0.101444278461934,0.342263584846614,0.0299209854562153,0.00655197214602716,0.00291564033770535,0.228585745920968,0.0443897851559131,0.00485785845789158,"grpNo>35"
+"No","None",1286.103617,"25-35",0.231346547873055,0.0433992224749338,0.436947009223752,0.0288085839354264,0,0.00473020985213511,0.224380950481379,0.0258253624054616,0.00456211375385627,"grpNo25-35"
+"No","Pill",2.608839,"<18",0,0,1,0,0,0,0,0,0,"grpNo<18"
+"No","Pill",4.445166,"18-20",0,0,0.425358018125757,0,0,0,0.574641981874243,0,0,"grpNo18-20"
+"No","Pill",34.133643,"20-25",0,0,0.761328786382397,0.0274189016390662,0,0.02329663434987,0.158185342244307,0.0297703353843597,0,"grpNo20-25"
+"No","Pill",41.816037,">35",0,0.199959934032008,0.54553357124684,0,0,0.0185051012844665,0.147410095318215,0.053944160227331,0.0346471378911397,"grpNo>35"
+"No","Pill",52.929725,"25-35",0,0.221710843953941,0.502061346436242,0.113012829747368,0,0.0563397221504552,0.0775337865443284,0.0293414711676662,0,"grpNo25-35"
+"No","IUD",2.339845,">35",0.205614901841789,0,0,0,0,0,0.399986751259165,0,0.394398346899047,"grpNo>35"
+"No","IUD",5.300697,"20-25",0.879360393548245,0,0,0,0,0,0.120639606451755,0,0,"grpNo20-25"
+"No","IUD",17.40484,"25-35",0.400447231919397,0,0.0687185288689813,0,0,0,0.301876661894048,0.228957577317574,0,"grpNo25-35"
+"No","Injectable",6.190851,"<18",0.683918899033429,0,0,0,0,0,0.316081100966571,0,0,"grpNo<18"
+"No","Injectable",6.228366,"18-20",0.618753136858046,0,0,0.138138317497719,0,0,0.104516497585402,0.138592048058833,0,"grpNo18-20"
+"No","Injectable",34.122848,"20-25",0.73250506522785,0.0907816076782337,0,0.071279718504153,0,0,0.0339214358660801,0.0715121727236836,0,"grpNo20-25"
+"No","Injectable",68.070351,">35",0.34717491026306,0.277854274616565,0,0,0,0,0.286026320034695,0.0889444950856798,0,"grpNo>35"
+"No","Injectable",100.395797,"25-35",0.557497820351981,0.122158619847403,0,0.0641977472423472,0,0,0.15947481347252,0.0903871005675666,0.00628389851818199,"grpNo25-35"
+"No","Condom",3.504164,">35",0,0,0.31947277581757,0,0,0,0.68052722418243,0,0,"grpNo>35"
+"No","Condom",13.892453,"25-35",0.170950371399493,0.462491757215231,0.122888592820865,0,0,0,0.0748109063244626,0,0.168858372239949,"grpNo25-35"
+"No","Condom",13.903033,"20-25",0.218155779390008,0,0.0605225492883459,0,0,0,0.215132050682754,0.168729873546297,0.337459747092595,"grpNo20-25"
+"No","Withdrawal",0.752835,">35",0,0,1,0,0,0,0,0,0,"grpNo>35"
```

Comment:
@mzimmermann-IDM it took me awhile to spot this, but this file seems to be corrupted somehow... it's missing several rows for withdrawal and IUD for example

Reply:
Ah yes ok I see. Unfortunately not corrupted, there's just no women in those groups so we don't have any data. Could we use the next closes age group for any row that's missing?